### PR TITLE
[1LP][RFR]Introducing parametrization in page per item test cases.

### PR DESCRIPTION
**Pull request by anurag03**
Purpose or Intent
=================
Introducing parametrization in page per item related test case to test complete options available on the setting page.

{{pytest: -v -k 'test_cloud_grid_page_per_item or test_cloud_tile_page_per_item or test_cloud_list_page_per_item'}}

### CODE REVIEW

**Comment by anurag03** [comment]
Changing the state from WIPTEST to RFR as there was no Orchestration Stacks available for the cloud provider let to failure of the test case.

File: cfme/tests/configure/test_visual_cloud.py [hunk 1]
```diff
@@ -19,13 +19,6 @@
               pytest.mark.usefixtures("openstack_provider")]
 
 # TODO When all of these classes have widgets and views use them in the tests
```
Comment (rlbabyuk) [inline cfme/tests/configure/test_visual_cloud.py]: Required: Please remove if it is already done, if not, then also move to list of classes

**Comment by psav** [comment]
Squash please

**Comment by anurag03** [comment]
@psav : done, changing the state from [1LP][WIP] to [1LP][RFR]

**Pull request merged**